forge-gui/src/main/java/forge/player/HumanCostDecision.java

```diff
@@ -667,6 +667,9 @@ public PaymentDecision visit(final CostPutCounter cost) {
         final int c = cost.getAbilityAmount(ability);
 
         if (cost.payCostFromSource()) {
+            if (ability.getDescription().contains("Fabricate") && !controller.confirmPayment(cost, Localizer.getInstance().getMessage("lblPutNTypeCounterOnTarget", String.valueOf(c), cost.getCounter().getName(), ability.getHostCard().getName()), ability)) {
```

Comment:
I don't like that check, can't we do something better?
Why is it necessary?

Reply:
By default GUI shouldn't ask (e.g. upticking Planeswakers)